### PR TITLE
initial server spring boot

### PR DESCRIPTION
**Pull request by doducphong**

### CODE REVIEW

**Review by dangquyitt — COMMENTED** [review]
@doducphong We only need to create the /server directory, not /serve/WedemyServer

File: server/WedemyServer/.dockerignore
Comment (dangquyitt) [inline server/WedemyServer/.dockerignore:1]: Remove this

Comment (doducphong) [reply server/WedemyServer/.dockerignore:1]: you mean is delete the pull request

Comment (dangquyitt) [reply server/WedemyServer/.dockerignore:1]: please note that this comment is in the file server/WedemyServer/.dockerignore, which means I want you to delete this file, not the entire PR

File: server/WedemyServer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>24.1.0</version>
+            <scope>compile</scope>
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <!--    If located in MAINLAND CHINA, UNCOMMENT THIS BLOCK -->
```
Comment (dangquyitt) [inline server/WedemyServer/pom.xml:163]: delete this comment

File: server/WedemyServer/src/main/java/com/utc2/onlinelearning/OnlineLearningApplication.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021. Davis Tibbz.  MIT License. Github: https://github.com/longwater1234
```
Comment (dangquyitt) [inline server/WedemyServer/src/main/java/com/utc2/onlinelearning/OnlineLearningApplication.java:2]: i think we should change this part

**Pull request merged**